substrate/cmd/kitctl/root.go

```diff
@@ -68,13 +68,15 @@ func newRootCmd(args []string) *cobra.Command {
 var options = &Options{}
 
 type Options struct {
-	file  string
-	debug bool
-	help  bool
+	file         string
+	debug        bool
+	help         bool
+	instanceType string
```

Comment:
I think this will also apply to delete command which doesn't make sense for the user and can get confusing. Right place for this IMO, for this flag is for subcommand `bootstrap`. please correct me if I am wrong.

Reply:
hmm makes sense..we can have a local flag on the bootstrap cmd

Reply:
have update it to local flag.